--- a/deps/v8/src/base/debug/stack_trace.h
+++ b/deps/v8/src/base/debug/stack_trace.h
@@ -17,1 +17,6 @@
-#include "src/base/build_config.h"
+#include "src/base/build_config.h"
+#if defined(__ANDROID__) && defined(__ANDROID_API__) && (__ANDROID_API__ < 33)
+#define backtrace(buffer, size)                 (0)
+#define backtrace_symbols(buffer, size)         ((char**)0)
+#define backtrace_symbols_fd(buffer, size, fd)  ((void)0)
+#endif
