diff --git a/deps/v8/src/base/debug/stack_trace_posix.cc b/deps/v8/src/base/debug/stack_trace_posix.cc
--- a/deps/v8/src/base/debug/stack_trace_posix.cc
+++ b/deps/v8/src/base/debug/stack_trace_posix.cc
@@ -375,1 +375,4 @@  // return values, we take no chance.
-  count_ = static_cast<size_t>(backtrace(trace_, arraysize(trace_)));
+// Android: backtrace() / backtrace_symbols() are only available in API 33+
+#if !defined(__ANDROID__) || (__ANDROID_API__ >= 33)
+  count_ = static_cast<size_t>(backtrace(trace_, arraysize(trace_)));
+#endif
